### PR TITLE
Update refs to IDR URL & name

### PR DESCRIPTION
**Pull request by hflynn**
The IDR is being published as 'Image Data Resource' and '-demo' is being removed from the URL - this PR updates the old blog posts.

Note there is no staging build for the blog, you need to build locally with jekyll, although proofing should be sufficient for these changes.

### CODE REVIEW

File: _posts/2015-12-09-omero-status.md [hunk 1]
```diff
@@ -99,3 +99,4 @@ they already do with images and tags. All of these new capabilities will start
 to appear in OMERO 5.3 (currently scheduled for Spring 2016) and in several
 point releases through 2016.
 
+[^1]: At time of publication, this was referred to as the 'Image Data Repository'.
```
Comment (rleigh-codelibre) [inline _posts/2015-12-09-omero-status.md]: "At the" ?

File: _posts/2016-01-26-bf-model-status.md [hunk 2]
```diff
@@ -91,3 +91,5 @@ provide the reference implementation for working with the file formats defined
 by the Open Microscopy Environment—OME-XML and OME-TIFF—in Java and C++
 and the new development cycle will allow us to get updates out to our users as
 quickly as possible.
+
+[^1]: At time of publication, this was referred to as the 'Image Data Repository'.
```
Comment (rleigh-codelibre) [inline _posts/2016-01-26-bf-model-status.md]: "At the"?

File: _posts/2016-07-05-eu-ref.md [hunk 3]
```diff
@@ -53,3 +53,5 @@ aspects of the modern, interconnected world that unites dedicated, talented
 individuals, regardless of race, religion, gender, sexual identity or national
 origin, with a mission to drive scientific insight and discovery for the
 benefit of all humankind.
+
+[^1]: At time of publication, this was referred to as the 'Image Data Repository'.
```
Comment (rleigh-codelibre) [inline _posts/2016-07-05-eu-ref.md]: "At the"?

**Comment by rleigh-codelibre** [comment]
Looks fine, other than a missing "the" in three places.

**Comment by hflynn** [comment]
@rleigh-codelibre thanks, changes pushed

**Comment by rleigh-codelibre** [comment]
Thanks.  All looks good to me.

**Pull request merged**